.gitattributes

```diff
@@ -13,3 +13,5 @@
 *.bat eol=crlf
 # The az script for Git Bash/Cygwin should be LF
 build_scripts/windows/scripts/az eol=lf
+# sh scripts should be LF
+*.sh eol=lf
```

Comment:
why need this? Will git applicaiton get conflict with it?

Reply:
I would like reuse the `scripts/ci/build.sh` to run on Git Bash. Without this setting, all files checked out on Windows will be converted to crlf and will have error when run with bash.